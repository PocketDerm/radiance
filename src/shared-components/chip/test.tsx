import React from 'react';
import renderer from 'react-test-renderer';

import { Chip } from './index';

describe('<Chip />', () => {
  describe('UI snapshots', () => {
    it('renders the correct css and text', () => {
      const component = renderer.create(
        <Chip status="success" text="Success" />,
      );

      const tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
