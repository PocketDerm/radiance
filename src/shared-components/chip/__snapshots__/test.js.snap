// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Chip /> UI snapshots renders the correct css and text 1`] = `
.emotion-2 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-radius: 1rem;
  border: 1px solid;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  height: 1.5rem;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  min-width: 2rem;
  background-color: #eaf2eb;
  border-color: #DAECDC;
}

.emotion-2 .emotion-1 {
  color: #317f3b;
}

.emotion-0 {
  color: #5C5876;
  font-size: 0.75rem;
  line-height: 1.67;
  font-weight: bold;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  text-transform: uppercase;
  -webkit-letter-spacing: 0.5px;
  -moz-letter-spacing: 0.5px;
  -ms-letter-spacing: 0.5px;
  letter-spacing: 0.5px;
  font-size: 10px;
  line-height: 1;
  padding: 0.5rem 12px;
  position: relative;
  top: 0.5px;
}

<div
  className="emotion-2 emotion-3"
>
  <span
    className="emotion-0 emotion-1"
  >
    Success
  </span>
</div>
`;
